### DISCUSSION TITLE
[ESP8266] Can other pins be used instead of D1/D2 for dscClockPin/dscReadPin?

### DISCUSSION THREAD

**Discussion by matomato13** [taligentx/dscKeybusInterface#293] [dscKeybusInterface Discussion] [answered]
I'm not sure if the D1/D2 pins are working correctly on my NodeMcu, I tried the Pushover file, I receive the Initializing and Connected push notifications but nothing else. I was wondering if I could change the pins to test.

I tried changing D1/D2 for D3/D4 and D5/D6 but I get nothing more than the two previously mentioned push notifications...

**Comment by matomato13** [comment, marked as answer]
My issue was that I'm using a PC1555 and the voltage is not ~12v but more like ~7-9v so I had to use 15k resistors instead of the 33k.

**Reply by matomato13** [reply]
Not really an answer since I'm still using D1/D2 but marking as solved.

**Reply by luk2009** [reply]
I use D6 and D7 and everything work perfect
